### DISCUSSION TITLE
does mesh agent supports and run in chrome OS

### DISCUSSION THREAD

**Discussion by a-imran99** [Ylianst/MeshCentral#4469] [Q&A] [answered]
Does mesh agent supports chrome/android OS and can it be connected and take control of?

**Comment by si458** [comment]
i dont believe you can control a chromeos device,
BUT you can use chrome os to control other devices as its all web based

**Reply by a-imran99** [reply]
@si458 Can you explain pls.

**Comment by krayon007** [comment, marked as answer]
If you refer to the [Readme](https://github.com/Ylianst/MeshAgent) in the agent repo, there is a section on ChromeOS. I outlined how to run the agent on ChromeOS. It involves putting the thing in developer mode and such. Another caveat is that remote desktop is not supported. Just terminal and files. 

You can also run the Android agent on ChromeOS, that method does not require developer mode.
